CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 message("💡 Starting UBC Formula Electric Generate")
 
 cmake_minimum_required(VERSION 3.24)
-set(CMAKE_SYSTEM_VERSION 1)
+# set(CMAKE_SYSTEM_VERSION 1) not necessarily required?
```

Comment:
?

Reply:
I removed it and it didn't change anything. Maybe check on linux?